internal/pkg/symnotify/symnotify.go

```diff
@@ -57,18 +54,18 @@ func (w *Watcher) EventTimeout(timeout time.Duration) (e Event, err error) {
 	case !ok:
 		return Event{}, io.EOF
 	case e.Op == Create:
-		log.Info("Create Event Detected for file.." ,"e.Name",e.Name)
+		log.V(2).Info("Create Event Detected for file.." ,"e.Name",e.Name)
 		if info, err := os.Lstat(e.Name); err == nil {
 			if isSymlink(info) {
 				_ = w.watcher.Add(e.Name)
 			}
 		}
 	case e.Op == Remove:
-		log.Info("Create Event Detected for file.." ,"e.Name",e.Name)
+		log.V(2).Info("Remove Event Detected for file.." ,"e.Name",e.Name)
 		w.watcher.Remove(e.Name)
 		w.watcher.Add(e.Name)
```

Comment:
```suggestion
```

We don't need to handle Remove, the w.watcher already knows it's been removed (it told us) and there's no reason to re-add (it will fail)

Reply:
@alanconway 
Without the below 
   case e.Op == Remove:
                log.V(2).Info("Remove Event Detected for file.." ,"e.Name",e.Name)
                w.watcher.Remove(e.Name)
                w.watcher.Add(e.Name)
I couldn't get metric values right. Will retest it by completely removing the above line of code.

Reply:
@alanconway working now by removing both the below treatment on the occasion of detected Remove event.
w.watcher.Remove(e.Name)
w.watcher.Add(e.Name)